docs/decisions/0002-text-editor-adr.md

```diff
@@ -0,0 +1,86 @@
+# Status
+
+Accepted
+
+# Context
+
+We seek to hoist the legacy text (HTML) text editor out of the monolith into a decoupled React application
+without reducing editor capabilities in the process.
+
+The legacy editor delegates the bulk of its operations to the tinyMCE editor
+
+
+
+# Decision set (each decision small)
+
+1. Rely on editor-wide Blockstore networking utilities for loading from, and saving to, XBlock content
```

Comment:
Important note. **Blockstore** is a system for storing versioned, reusable educational content for Open edX. Blockstore, however, is presently only used as the storage method for V1 and V2 content libraries. Presently We use the [modulestore](https://github.com/openedx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/docs/overview.rst#overview-of-the-edx-modulestores) for our backend storage needs, but all of this is superflous-- what we need to tell the reader here is that we use the CMS' **xblock rest api**. Blockstore isn't being used at all, and doesn't point readers in the direction of the abstraction which provides information that is relevant to them.

Reply:
so, maybe "Rely on editor-wide networking utilities for loading from, and saving to, XBlock content"? (One word, Blockstore, deleted)

Reply:
Per our out-of-band discussions, discussion on XBlock network support will be in a separate ADR with cross-editor scope.